doc/cygnus-ngsi-ld/installation_and_administration_guide/README.md

```diff
@@ -5,15 +5,13 @@
     * [Installation via docker](./install_with_docker.md)
     * [Installation from sources](./install_from_sources.md)
 * Configuration:
-    * [NGSI agent configuration](./ngsi_agent_conf.md)
-    * [log4j configuration](../../installation_and_administration_guide/log4j_conf.md)
```

Comment:
Why this removal? log4j is not being used by cygnus-ngsi-ld?

Reply:
We don't have yet and specific configuration for log4j

Reply:
If you plan to add it in the future, then I'd suggest not removing the item, but "disabling" the link. I mean, something like this:

```
     * log4j configuration (FIXME: we don't have yet and specific configuration for log4j but it will be done in the future)
```

Reply:
I agree, included in 846fae7